src/app/movies/[movieId]/page.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+async function getMovie(id: any) {
+  const res = await fetch(`https://plankton-app-xhkom.ondigitalocean.app/api/movies/${id}`);
+  return res.json();
+}
+
+export default async function movieInfo({ params: { movieId } }) {
+  console.log(`hej ${movieId}`);
+  const movieData = getMovie(movieId);
+  const [movie] = await Promise.all([movieData]);
+  console.log(movie);
+  return (
+    <>
+      <main className="sec">
```

Comment:
main is in layout.tsx and is not needed here. As it is now it is main element inside main element.